pkgs/development/python-modules/pybluez/default.nix

```
@@ -2,11 +2,14 @@
 , buildPythonPackage
 , fetchFromGitHub
 , pkgs
+, isPy3k
 }:
 
 buildPythonPackage rec {
   version = "unstable-20160819";
   pname = "pybluez";
+  # requires use2to3, which is no longer supported in setuptools>58
+  disabled = isPy3k;
```

```suggestion
  disabled = true;
```
we are always in python3 in this directory.

---

How so? I can still (try and) build `python2Packages.pybluez`.

---

Ah, you are right. I thought python2Packages wasn't overlayed https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python2-packages.nix#L1-L3